var/spack/repos/builtin/packages/zoltan/package.py

```diff
@@ -88,9 +88,15 @@ def install(self, spec, prefix):
             config_args.append('CXX={0}'.format(spec['mpi'].mpicxx))
             config_args.append('FC={0}'.format(spec['mpi'].mpifc))
 
-            mpi_libs = ' -l'.join(self.get_mpi_libs())
             config_args.append('--with-mpi={0}'.format(spec['mpi'].prefix))
-            config_args.append('--with-mpi-libs=-l{0}'.format(mpi_libs))
+
+            mpi_libs = self.get_mpi_libs()
+
+            if mpi_libs:
+                mpi_libs = ' -l'.join(mpi_libs)
+                config_args.append('--with-mpi-libs=-l{0}'.format(mpi_libs))
+            else:
+                config_args.append('--with-mpi-libs= ')
```

Comment:
maybe add a short comment above why we do it (i.e. external MPI on bg-q with no libs in `lib` folder)